### ISSUE TITLE
[feature request] Math keypad

### ISSUE THREAD

**Issue by ghost** [Julow/Unexpected-Keyboard#133]
It would be nice to have a dedicated keypad for math symbols, other than the symbols present in the [alphabetic](https://f-droid.org/repo/juloo.keyboard2/en-US/phoneScreenshots/1.png) and [numeric keypad](https://f-droid.org/repo/juloo.keyboard2/en-US/phoneScreenshots/6.png).

https://math.typeit.org/ might be a good reference

**Comment by Julow** [comment]
Good idea. Would you be interested in making the layout ? A layout is described like this: [numeric](https://github.com/Julow/Unexpected-Keyboard/blob/master/res/xml/numeric.xml)

**Comment by Julow** [comment]
The [greekmath](https://github.com/Julow/Unexpected-Keyboard/blob/master/res/xml/greekmath.xml) pane has just been added. It still lacks many of the symbols that are show on https://math.typeit.org/, contributions welcome.

If you want to try it out, you can find a debug build here (it will be a separate app that you need to enable again in the virtual keyboards setting): https://github.com/Julow/Unexpected-Keyboard/actions/runs/2641578170
